docs/src/examples/solvers/DC-DC converter.jl

```diff
@@ -160,6 +160,7 @@ control_trajectory = ST.get_closed_loop_trajectory(
 
 fig = plot(; aspect_ratio = :equal);
 plot!(concrete_system.X; label = "", color = :grey);
+plot!(uninvariant_set; color = :black, label = "Uninvariant set")
```

Comment:
"Uninvariant" ?

Reply:
What does that mean ?

Reply:
This is the optimizer for the "safety problem". So invariant_set is the largest invariant set we manage to compute by solving the fixed point equation. And uninvariant_set is the remaining, i.e., the domain minus the invariant set. Is there a better name ;) ?

Reply:
done ;)